src/util/loanSearch/queryParamUtils.js

```diff
@@ -23,10 +23,18 @@ export function hasExcludedQueryParams(query) {
 	// Handle temporary query param exclusions
 	const excludedParams = [
 		'activity',
+		'city_state',
+		'defaultRate',
+		// We don't yet officially support this query param, but will soon
+		// Dropping it doesn't have any ill affect on the most popular categories
+		// If problems are reported we can uncomment this one to redirect to legacy
+		// 'distributionModel',
```

Comment:
As noted, I've not seen any big issues with this when landing on the flss version of lend/filter. Typically the combination of query params produces a proper loan search state. ie. A US loan search includes query params with the countryIsoCode so the result is correct.